### PR TITLE
Feature/pelagos 3538 fix css and search results inclusive

### PR DESCRIPTION
**Pull request by praneethpr**

### CODE REVIEW

File: src/Pelagos/Bundle/AppBundle/Resources/views/Search/default.html.twig [hunk 1]
```diff
@@ -3,19 +3,44 @@
 {% block head %}
     {{ add_css (
```
Comment (mickel1138) [inline src/Pelagos/Bundle/AppBundle/Resources/views/Search/default.html.twig]: By overwriting the head, you basically remove all the head stuff from the template.

Just use stylesheet and javascipts blocks to add stuff, prefix each block with {{ parent() }}

Comment (mickel1138) [reply src/Pelagos/Bundle/AppBundle/Resources/views/Search/default.html.twig]: I forgot, this is not the new framework yet.

File: src/Pelagos/Bundle/AppBundle/Resources/views/Search/default.html.twig [hunk 2]
```diff
@@ -3,19 +3,44 @@
...
         ]
     ) }}
     {% stylesheets
         '@PelagosAppBundle/Resources/public/css/search.css' %}
         {{ add_css(asset_url) }}
     {% endstylesheets %}
+
+    {{ add_js (
+        [
+            '//code.jquery.com/jquery-2.1.3.min.js',
```
Comment (mickel1138) [inline src/Pelagos/Bundle/AppBundle/Resources/views/Search/default.html.twig]: You shouldn't have to include jQuery right?

Comment (praneethpr) [reply src/Pelagos/Bundle/AppBundle/Resources/views/Search/default.html.twig]: The pagination plugin requires at least that version of jQuery, I assumed that we have an older version of jQuery

Comment (mickel1138) [reply src/Pelagos/Bundle/AppBundle/Resources/views/Search/default.html.twig]: I removed this line, and it seems to still work...

Comment (praneethpr) [reply src/Pelagos/Bundle/AppBundle/Resources/views/Search/default.html.twig]: Yes, i tried it too. Will remove it 👍

**Comment by mickel1138** [comment]
On subsequent searches, the paginator links do not get updated.

**Comment by mickel1138** [comment]
also the search bar doesn't update, so I think that's were the issue is.

**Comment by praneethpr** [comment]
> On subsequent searches, the paginator links do not get updated.

Good find. 👍

**Pull request merged**